src/well_known_did_config.rs

```diff
@@ -180,8 +179,8 @@ impl LinkedDid {
 
 pub fn create_well_known_did_config(
     cfg: &config::WellKnownDidConfig,
-) -> Result<WellKnownDidConfig, Box<dyn std::error::Error>> {
-    let pair = sp_core::sr25519::Pair::from_string_with_seed(&cfg.seed, None)?.0;
+) -> anyhow::Result<WellKnownDidConfig> {
```

Comment:
What is anyhow?

Reply:
A create that makes error handling easy. You have lots of different error types. Anyhow merges them all into one. This makes sense for CLI tools where you don't handle the specific error and most likely just display that to the user. You can't really check the error in your code anymore when you use anyhow. That's why it's called anyhow. You don't care about the specific error

Reply:
That is cool